board/opentrons/ot2/rootfs-overlay/etc/fw_env.config

```diff
@@ -0,0 +1,7 @@
+# This is the configuration file for fw_(printenv,setenv) which is used to
+# switch the boot partition used by the system. This requires the FAT boot
+# partition to be mounted to /mnt/boot, which is _not_ done automatically -
+# the boot partition should only ever be mounted temporarily to avoid accidents
+
+# "MTD device" name  Offset  Size
+/mnt/boot/uboot.env  0x0     0x4000
```

Comment:
Is no newline at EOF going to cause any issues?

Reply:
No but I'll add it anyway